pyproject.toml

```diff
@@ -222,6 +222,9 @@ environment-pass = ["SKIMAGE_LINK_FLAGS", "PIP_EXTRA_INDEX_URL", "COLUMNS"]
 build-verbosity = 0
 enable="cpython-freethreading"
 
+[[tool.cibuildwheel.config-settings]]
+setup-args = ["-Dinclude-v2=false"]
```

Comment:
I'm not clear right now how to override this setting. For example for the nightly wheel publishing we might want to include the namespace?

Maybe we can document that some more?